octobot_trading/personal_data/orders/order.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, trader, side=None):
         self.simulated = trader.simulate
 
         self.logger_name = None
-        self.order_id = trader.parse_order_id(None)
-        self.shared_signal_order_id = str(uuid.uuid4())
+        self.order_id = trader.parse_order_id(None)      # order id: given by the exchange local to the user account
+        self.shared_signal_order_id = str(uuid.uuid4())  # shared order id: kept through instances and trading signals
```

Comment:
I think we should name it "internal_order_id" or something like this because when I first read about it I expected it to be an order id shared with someone else. As I understand, it could be shared when using copy trading but it may also not be shared. I would prefer to name it differently as it is not always shared with others

Reply:
right, i'll change it.
what about `bot_order_id` ? as it will be shared, it's not really internal